libs/blocks/section-metadata/section-metadata.css

```diff
@@ -204,6 +204,11 @@
 }
 
 .section.sticky-bottom.promo-sticky-section {
+  background: none;
+  z-index: 4; 
+}
+
+.section.sticky-bottom.promo-sticky-section.popup {
   background: none;
   z-index: 40000;
```

Comment:
I'd say that the real issue here is that we're using this `40000` value. Removing this might be a better path forward, we should never have `z-index` values larger than `10`

Reply:
And that removal also fixes the issues reported by QA in the story.

Reply:
Yes, agreed. However, this specific change of adjusting the z-index to 40,000 was made a few weeks ago in reference to ticket [MWPW-155694](https://jira.corp.adobe.com/browse/MWPW-155694). @Brandon32 might be able to provide further insights on this.